codegen/templates/endpoint.tmpl

```diff
@@ -50,20 +52,25 @@ func Handle{{title .Name}}Request(
 	{{end}}
 
 	headers := map[string]string{}
+	// TODO(sindelar): Add optional headers in addition to required.
+	for _, k := range {{.Headers | printf "%#v" }} {
```

Comment:
We want to parse all the headers here.

We do not want to *propagate* all the headers.